docs/src/selenium-grid.md

```diff
@@ -160,24 +160,24 @@ Here is an example of running selenium hub and a single selenium node, and conne
 First start the hub container and one or more node containers.
 
 ```bash
-docker run -d -p 4442-4444:4442-4444 --name selenium-hub selenium/hub:4.3.0-20220726
+docker run -d -p 4442-4444:4442-4444 --name selenium-hub selenium/hub:latest
 docker run -d -p 5555:5555 \
     --shm-size="2g" \
     -e SE_EVENT_BUS_HOST=<selenium-hub-ip> \
     -e SE_EVENT_BUS_PUBLISH_PORT=4442 \
     -e SE_EVENT_BUS_SUBSCRIBE_PORT=4443 \
     -e SE_NODE_GRID_URL="http://<selenium-hub-ip>:4444"
-    selenium/node-chrome:4.3.0-20220726
+    selenium/node-chrome:latest
```

Comment:
We usually recommend running with exact version labels to avoid unexpected issues related to environment changes. Maybe we can simply update the tag to a newer version here?

Reply:
Yes, I updated it to the latest tag version. I also added a reference link to the tagging convention for someone who wants to understand more.